packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1227,9 +1235,13 @@ class _ImageState extends State<Image> with WidgetsBindingObserver {
     _isListeningToStream = true;
   }
 
-  void _stopListeningToStream() {
+  void _stopListeningToStream([bool createPassiveListener = false]) {
+
     if (!_isListeningToStream)
       return;
+    // if (createPassiveListener) {
+    //   _imageStream!.completer?.addPassiveListener(null);
+    // }
```

Comment:
?

Reply:
haha whups :)  Thought I needed this at one point but I do not.

Reply:
Oh. I think I do need something like this. I'll fix it and add the relevant test.

The problem right now is that if the TickerMode goes off, we could inadvertently dispose of the image stream when we stop listening to it. We'll have to add a passive listener and remove it.